spec/factories/in_person_enrollments.rb

```diff
@@ -2,7 +2,7 @@
   factory :in_person_enrollment do
     user { association :user, :signed_up }
     profile { association :profile, user: user }
-
+    unique_id { Faker::Number.number(digits: 18) }
```

Comment:
```suggestion
    unique_id { Faker::Number.hexadecimal(digits: 18) }
```

to be more consistent with the strings the app will generate in reality

Reply:
Done